cirq/circuits/circuit.py

```diff
@@ -1250,7 +1255,8 @@ def _draw_moment_in_diagram(moment: Moment,
                             use_unicode_characters: bool,
                             qubit_map: Dict[ops.QubitId, int],
                             out_diagram: TextDiagramDrawer,
-                            precision: Optional[int]):
+                            precision: Optional[int],
+                            is_transposed: bool):
```

Comment:
is_transposed argument is not used?

Reply:
Removed. Was a leftover from experimenting.